apps/studio/components/interfaces/Settings/Database/NetworkRestrictions/NetworkRestrictions.tsx

```diff
@@ -101,187 +109,164 @@ const NetworkRestrictions = () => {
-                      disabled={!canUpdateNetworkRestrictions}
-                      onClick={setIsAllowingAll}
-                    />
-                    <DisallowAllAccessButton
-                      disabled={!canUpdateNetworkRestrictions}
-                      onClick={setIsDisallowingAll}
-                    />
+                }
+              >
+                <AllowAllAccessButton
+                  disabled={!canUpdateNetworkRestrictions}
+                  onClick={setIsAllowingAll}
+                />
+              </FormLayout>
+            </CardContent>
+          </Card>
+        ) : (
+          <Card>
+            <CardHeader>Only the following IP addresses have access to your database</CardHeader>
+            {restrictedIps.map((ip) => {
+              return (
+                <CardContent className="flex items-center justify-between gap-4">
+                  <Globe size={16} className="text-foreground-light" />
+                  <div className="w-full flex items-center gap-4">
+                    <Badge>{ipv4Restrictions.includes(ip) ? 'IPv4' : 'IPv6'}</Badge>
+                    <p className="text-sm font-mono">{ip}</p>
                   </div>
-                </div>
-                {restrictedIps.map((ip) => {
-                  return (
-                    <div key={ip} className="px-8 py-4 flex items-center justify-between">
-                      <div className="flex items-center space-x-5">
-                        <Globe size={16} className="text-foreground-light" />
-                        <Badge>{ipv4Restrictions.includes(ip) ? 'IPv4' : 'IPv6'}</Badge>
-                        <p className="text-sm font-mono">{ip}</p>
-                      </div>
-                      <Button type="default" onClick={() => setSelectedRestrictionToRemove(ip)}>
-                        Remove
-                      </Button>
-                    </div>
-                  )
-                })}
-              </div>
-            )}
-          </FormPanel>
+                  <Button type="default" onClick={() => setSelectedRestrictionToRemove(ip)}>
+                    Remove
+                  </Button>
+                </CardContent>
+              )
+            })}
+            <CardContent className="text-sm text-foreground-light space-y-1">
+              <p>
+                You may remove all of them to allow all IP addresses to have access to your database
+              </p>
+              <p>Note: Restrictions only apply to your database, and not to Supabase services</p>
+            </CardContent>
+            <CardFooter className="justify-end">
+              <AllowAllAccessButton
+                disabled={!canUpdateNetworkRestrictions}
+                onClick={setIsAllowingAll}
+              />
+              <DisallowAllAccessButton
+                disabled={!canUpdateNetworkRestrictions}
+                onClick={setIsDisallowingAll}
+              />
+            </CardFooter>
+          </Card>
```

Comment:
here I felt it made more sense to re-order this so that the IP list comes directly after the header, and the help text always appears near the `Allow All` / `Disallow All` actions